src/usr/local/www/diag_backup.php

```diff
@@ -277,7 +276,7 @@ function check_and_returnif_section_exists($section) {
 								$savemsg = gettext("The configuration area has been restored. The firewall may need to be rebooted.");
 							}
 						}
-					} else {
+					} else if (!$input_errors) {
```

Comment:
This should be `elseif` ([ref](https://docs.netgate.com/pfsense/en/latest/development/developer-style-guide.html#general-rules))

Reply:
fixed